Android insecure deserialization (#1273)
microsoft/appcenter-sdk-android: CHANGELOG.md

  - Accounts in any Azure AD directory.
  - Accounts in any Azure AD directory and personal Microsoft accounts.

### App Center Crashes

> * **[Breaking change]** Remove insecure implementation of `ErrorReport.getThrowable` and `ErrorReport.setThrowable` (now marked as deprecated and always returns `null`). Provide `ErrorReport.getStackTrace` and `ErrorReport.setStackTrace` as an alternative.

Reviewer: There is extra `)` in @guperrot 's suggestion

```suggestion
* **[Breaking change]** Remove insecure implementation of `ErrorReport.getThrowable()` that marked as deprecated now and always returns `null`. Provide `ErrorReport.getStackTrace()` as an alternative.
```

> * **[Breaking change]** Remove insecure implementation of `ErrorReport.getThrowable` and `ErrorReport.setThrowable` (now marked as deprecated and always returns `null`). Provide `ErrorReport.getStackTrace` and `ErrorReport.setStackTrace` as an alternative.

Reviewer 2: Yet another suggestion for this

```suggestion
* **[Behavior change]** Deprecate and remove insecure implementation of `ErrorReport.getThrowable()`, which now always returns `null`. Use the new `ErrorReport.getStackTrace()` as an alternative.
```

> * **[Breaking change]** Remove insecure implementation of `ErrorReport.getThrowable` and `ErrorReport.setThrowable` (now marked as deprecated and always returns `null`). Provide `ErrorReport.getStackTrace` and `ErrorReport.setStackTrace` as an alternative.

Author: thanks @guperrot @MatkovIvan @achocron for your suggestion!
